index.js

```diff
@@ -22,6 +27,7 @@ type Questionnaire {
 type Section {
   id: ID!
   title: String!
+  titleNew: Markup!
```

Comment:
How temporary is this field? It would benefit from a more descriptive name if it's going to be around for a while.

Reply:
Just temporary until author has migrated to this

Reply:
Gone with different approach in latest commit. This approach does not need a new Markup type. Instead opting for a field arg, which I've named `format`, defaulting to HTML, but allows us to select plaintext by passing in `(format: Plaintext)` into the query.